docs/source/tutorials/muon_GUI_course/the_tabs_home.rst

```diff
@@ -0,0 +1,70 @@
+.. _the_tabs_home:
+
+===============
+The Tabs - Home
+===============
+
+.. index:: The Tabs - Home
+
+.. contents:: Table of Contents
+  :local:
+
+Home
+====
+
+When launched, the Muon Analysis GUI defaults to the Home tab. This tab allows: 
+
+* data files to be loaded
+* run information to be viewed
+* detector groupings to be selected
+* type of plot to be selected
+
+Loading a Workspace
+-------------------
+
+To load a file either: **Browse** or **Load Current Run** or simply type a run number (assuming you have defined the directory(ies) in which your files are stored)
+When typing a run number, or using the Load Current Run option, first select the desired instrument from the dropdown list. 
+ 
+To demonstrate: 
+1. Select MUSR in the instrument drop-down menu
+2. Type run number 24563 in the Loading section and press enter, note this can only be done if the correct reference material folder was selected in :ref:`getting_started`. This process shown in Figure 16.
```

Comment:
```suggestion
2. Type run number '24563' in the Loading section and press enter, note this can only be done if the correct reference material folder was selected in :ref:`getting_started`. This process shown in Figure 16.
```